book/core-concepts/styles.md

```diff
@@ -1,12 +1,12 @@
 # Styles
 
-React Native allows you to style components using CSS properties as inline styles. For layout only the flexbox module and absolute positioning is available. The style properties are split into five categories:
+React Native allows you to style components using limited CSS properties as inline styles. For layout only the flexbox module and absolute positioning is available. The [style properties](https://facebook.github.io/react-native/releases/next/docs/style.html#supported-properties) are split into five categories:
```

Comment:
> using ~limited~ **a subset of** CSS properties

Suggestion: Clarify text a little and disambiguate what is actually limited (i.e. implying limited subset vs the CSS properties themselves being limited).